tools/bumbershoot/custom_pro_db/customProDB.R

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env Rscript
+
+initial.options <- commandArgs(trailingOnly = FALSE)
+script_parent_dir <- dirname(sub("--file=", "", initial.options[grep("--file=", initial.options)]))
+
+## begin warning handler
+withCallingHandlers({
+
+library(methods) # Because Rscript does not always do this
+
+options('useFancyQuotes' = FALSE)
+
+suppressPackageStartupMessages(library("optparse"))
+suppressPackageStartupMessages(library("RGalaxy"))
+
+
+option_list <- list()
+
+option_list$bam <- make_option('--bam', type='character')
```

Comment:
It's easier to test the R wrapper in RStudio with its own parameter parsing. Also, RGalaxy can autogenerate these wrappers from roxygen2 comments. Although I did have to customize it quite a bit.